### DISCUSSION TITLE
function parameters are shown next to the function call

### DISCUSSION THREAD

**Discussion by benzwt** [neoclide/coc.nvim#3992] [Q&A] [answered]
When I edit my c-file with neovim, function parameters are shown next to the function call, but not with the vim.
```
memcpy(*mem, buffer, offset); ‣dest: *mem ‣src: buffer ‣n: offset
```
I don't need this hint. How do I disable it ?
thanks!

**Comment by xiyaowong** [comment, marked as answer]
If you are using `coc-clangd,` try setting `clangd.inlayHints.enable` to `false`

**Reply by benzwt** [reply]
thank you !
